src/main/java/frc/robot/RobotContainer.java

```diff
@@ -453,10 +465,10 @@ public void disabledPeriodic() {
             nextAuto = "Waiting";
         }
 
-        if (!nextAuto.equals(m_autoCommandName)) {
-            bagManager.startBag(); // Start recording
-            m_autoCommandName = nextAuto;
-        }
+        // if (!nextAuto.equals(m_autoCommandName)) {
+        // bagManager.startBag(); // Start recording
```

Comment:
This is the only line that should be commented out. You want to keep the rest so auto selection works.